src/main/java/org/tron/common/utils/TransactionUtils.java

```diff
@@ -164,13 +167,10 @@ public static boolean validTransaction(Transaction signedTransaction) {
     return true;
   }
 
-  public static Transaction sign(Transaction transaction, ECKey myKey) {
+  public static Transaction sign(Transaction transaction, SignInterface myKey) {
     Transaction.Builder transactionBuilderSigned = transaction.toBuilder();
     byte[] hash = Sha256Hash.hash(transaction.getRawData().toByteArray());

```

Comment:
This code is not right. When it is ECKey, it Sha256Hash; when it is SM2, it is SM3